### ISSUE TITLE
Allow transcoding of multimedia content.

### ISSUE THREAD

**Issue by JulioNahum** [jarnedemeulemeester/findroid#43]
**The ability to choose the quality of playback when choosing a movie or series is urgently needed.**

**I think this feature would be the most essential since it would be the first jellyfin client for android that allowed content transcoding.** _I AM GRATEFUL FOR YOUR WORK and I WOULD LIKE YOU TO INCLUDE IT AS SOON AS POSSIBLE._ 

**I know we all have our lives and work, and you do too, but you have shown that you have done an excellent job creating this client. And I hope this client is the official Jellyfin for mobile needs. With this feature, the application would level up and be above EMBY.**

**Comment by SmearyThrower** [comment]
any chance this ever coming to findriod🤞🏻

**Comment by jramseygreen** [comment]
I, and many other jellyfin users I know, love this project but find the lack of transcoding a complete deal breaker. I really hope it's supported soon and this will become our jellyfin daily drive client!

**Comment by isZYKerman** [comment]
I really need the transcode feature. When I commute, my network becomes so poor that it cannot support high bitrate video. ☹️ I really want to see this feature in the app as soon as possible.

**Comment by mattalxndr** [comment]
Yeah hopefully this feature makes the list at some point. It's a great player though, when I'm unmetered.

**Comment by BillCipher-exe** [comment]
Yes please add this feature

**Comment by Bhuemann** [comment]
Yes I don't need any more features but this one

**Comment by KdntNinja** [comment]
This would also be useful as my home network doesn't have a high upload speed so streaming outside doesn't work without transcoding.

**Comment by l0stpt** [comment]
Is this going to happen in the future? 

Would be really cool since the official android jellyfin client can't play hevc files, only the web player but the web player sucks.

Also thanks for this amazing software I love it and use it daily.
